Assets/HoloToolkit/Input/Scripts/OnGazeEnterEvent.cs

```diff
@@ -1,21 +0,0 @@
-﻿using UnityEngine;
```

Comment:
Bare in mind that some people are happy to use the inspector but they're not coders so for simple things being able to implement a behaviour on gaze events straight from the editor is extremely useful and powerful !
For just gaze events, the performance hit is really negligible.
Anyway removing those files isn't related to your change, you can keep the current scripts by just updating them to the new design pattern (or even don't touch them as the old SendMessage way would have to stick around for a while)